pkg/utils/slices.go

```diff
@@ -0,0 +1,16 @@
+package utils
+
+import (
+	"strings"
+)
+
+func Intersect(this, that []string) bool {
```

Comment:
Can we move that out of `utils` package? I think it's better to have packages with more meaningful names. Here, I think we should go with `slices` or `sliceutil`, or something else 🤔

BTW It would be good to have a simple unit test for that.

Reply:
So, I completely agree with your thinking about `utils` packages. I don't like them.

In this case, I was going for the principle of least surprise. There is similar/general code in utils so I moved it there.

Reply:
I understand 👍 If you could move it out of there that'd be great - let's reduce the functions from there instead of adding new ones 🙂